.github/workflows/matrix.py

```diff
@@ -42,9 +42,7 @@
 ]
 
 # List of python versions to use for tests
-python_test_list = python_release_list + [
-    '3.10'
-]
```

Comment:
I know it would be weird Python to write, but I'd just delete the line with 3.10 on it and leave the list addition in place, since we know we'll need the list addition back in a couple of months. It might save someone having to figure out how matrix.py is supposed to work later.